CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 76.1.0
+
+Add additional snapshot tests and mark outdated theming API as deprecated.
```

Comment:
Tip: I don't know that we should call these APIs "outdated", we are just moving toward a newer approach.  Maybe the more neutral, "...and marking Dialogs Themer APIs as deprecated." would work?

Reply:
I think the newer approach validates calling the old API outdated, but I can definitely see the negative connotation.
s/outdated/previous/